layouts/partials/footer.html

```diff
@@ -18,8 +18,9 @@
 
     <div class="footer-copyright text-sm text-center text-gray-400 mt-4">
       {{- $copyright := (T "copyright" .) | default .Site.Copyright | default .Site.Title }}
-      &#169; {{ .Site.Params.copyrightYear }}-{{ dateFormat "2006" now.UTC }} {{ $copyright }}
-    </div>
+      {{- $currentYear := dateFormat "2006" now.UTC }}
+      &#169; {{ if lt .Site.Params.copyrightYear $currentYear }}{{ .Site.Params.copyrightYear }}-{{ end }}{{ $currentYear }} {{ $copyright }}
+    </div$>
```

Comment:
You have a stray `$` in the closing `</div>`

Reply:
Oops! Fixed.